src/include/duckdb/planner/binder.hpp

```diff
@@ -88,6 +88,8 @@ class Binder : public std::enable_shared_from_this<Binder> {
 	string alias;
 	//! Macro parameter bindings (if any)
 	MacroBinding *macro_binding = nullptr;
+	//! The intermediate lambda bindings to bind nested lambdas
+	vector<LambdaBinding> *lambda_bindings = nullptr;
```

Comment:
macro_bindings and lambda_bindings seem quite similar, can we unify them?

Reply:
I agree that `MacroBinding` and `LambdaBinding` should be unified to something like a `DummyBinding`. But I would not unify `macro_binding` and `lambda_bindings` because these are different things from a logical point of view. Also, we would use a vector for the macro bindings, when there can never be more than one.